websites/D/dアニメストア/metadata.json

```diff
@@ -22,4 +22,4 @@
 		"video",
 		"anime"
 	]
-}
\ No newline at end of file
+}
```

Comment:
If you can, remove the EOF newline character. (run `yarn format`)

Reply:
Doesn't hugely matter, SeMiD fixes these things